hbase-server/src/main/resources/hbase-webapps/master/table.jsp

```diff
@@ -387,68 +388,93 @@ if (fqtn != null && master.isInitialized()) {
     }
   }
 %>
-<table class="table table-striped">
-  <tr>
-    <th>RegionName</th>
-    <th>Start Key</th>
-    <th>End Key</th>
-    <th>Replica ID</th>
-    <th>RegionState</th>
-    <th>ServerName</th>
-  </tr>
-<%
-  final boolean metaScanHasMore;
-  byte[] lastRow = null;
-  try (final MetaBrowser.Results results = metaBrowser.getResults()) {
-    for (final RegionReplicaInfo regionReplicaInfo : results) {
-      lastRow = Optional.ofNullable(regionReplicaInfo)
-        .map(RegionReplicaInfo::getRow)
-        .orElse(null);
-      if (regionReplicaInfo == null) {
-%>
-  <tr>
-    <td colspan="6">Null result</td>
-  </tr>
-<%
-      continue;
-    }
+<div style="overflow-x: auto">
+  <table class="table table-striped nowrap">
+    <tr>
+      <th title="Region name">RegionName</th>
+      <th title="The startKey of this region">Start Key</th>
+      <th title="The endKey of this region">End Key</th>
+      <th title="Region replica id">Replica ID</th>
+      <th title="State of the region while undergoing transitions">RegionState</th>
+      <th title="Server hosting this region replica, stored in info:server column">Server</th>
+      <th title="The seqNum for the region at the time the server opened this region replica">Sequence Number</th>
+      <th title="Server where the region is transitioning on, stored in info:sn column">Target Server</th>
```

Comment:
s/Server where the region is transitioning on/The server to which the region is transiting/